examples/libjson-rpc-cpp/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
> Shouldn't I merge my libjson-rcp-cpp PR first to test it?

Chicken/egg problem. You should create such `.travis.yml`/`appveyor.yml` configuration so pull request to ruslo/hunter look green. In this case pull request will change RED (not working) state to RED (not working) state, so it's fine.

> feel free to delete this PR and do it your way

Please send pull request with excluded `.travis.yml`/`appveyor.yml` toolchains if you want https://github.com/ruslo/hunter/pull/790 to be merged.

> I just wanted to start some tests

You can do it in your fork actually, no need to open pull request. Just register your fork in Travis and AppVeyor.